src/templates/index.pug

```diff
@@ -8,6 +14,9 @@ body.layout
     // Page header
     nav.layout__header#header
         a.layout__header__menu(class=['logo'])="C++ Tour"
+        div
```

Comment:
Assign class `layout__header__icons` to this div.
Icons themself should use independent CSS that will be stored in common.sss

Reply:
Do you want me to change the class `layout__icons` to simply `icons` as well? And put it into common.sss?

Reply:
No.
Your structure should be something like:
```
div.layout__header__icons
    a.bk-image
```
Link itself shouldn't be aware of its placement and etc.

Reply:
Ok. so just adding the class then. The css stays the same.